libs/dh-app/core/feature-shell/src/lib/shell/shell.component.html

```diff
@@ -0,0 +1,12 @@
+<watt-shell>
+  <ng-container sidenav>
+    <p>Sidenav</p>
+  </ng-container>
+
+  <ng-container toolbar>
```

Comment:
suggestion: use a `watt-shell` prefix to be explicit about cohesive parts

```suggestion
  <ng-container watt-shell-toolbar>
```